SimplenoteUITests/NoteList.swift

```diff
@@ -14,13 +14,27 @@ class NoteList {
     }
 
     class func isAllNotesListOpen() -> Bool {
-        return app.navigationBars[UID.NavBar.allNotes].exists
+        let navBar = app.navigationBars[UID.NavBar.allNotes]
+        guard navBar.exists else { return false}
+        return navBar.frame.minX == 0.0
     }
 
     class func isNoteListOpen(forTag tag: String) -> Bool {
         return app.navigationBars[tag].exists
     }
 
+    class func getNoteCell(_ noteName: String) -> XCUIElement {
+        return Table.getCell(label: noteName)
+    }
+
+    class func isNotePresent(_ noteName: String) -> Bool {
+        return NoteList.getNoteCell(noteName).exists
+    }
+
+    class func getNoteCellHeight(_ noteName: String) -> CGFloat {
+        return NoteList.getNoteCell(noteName).frame.height
+    }
+
```

Comment:
These three methods still use `noteName` String param but not `NoteData` type because one of the methods is used in older methods, that accepts the String param too.

I'll change that with the next refactor commit, otherwise there will be too much noise.

Reply:
> I'll change that with the next refactor commit, otherwise there will be too much noise.

Excellent idea 👍